catalog_integrations_api_tools/sftp.py

```diff
@@ -30,14 +53,38 @@ def get_all_files(
         Returns:
             str: Name of directory the files were downloaded to.
         """
-        pass
+        products_pattern_compiled = re.compile(
+            rf"{file_prefix}_products_\d+\.csv(\.gz)?", flags=re.I
+        )
+        stockprices_pattern_compiled = re.compile(
+            rf"{file_prefix}_stock-prices_\d+\.csv(\.gz)?", flags=re.I
+        )
+
+        kwargs = self._get_sftp_credentials()
+        with pysftp.Connection(**kwargs) as sftp:
+            with sftp.cd(remote_directory):
+                for filename in sftp.listdir():
+                    if re.match(products_pattern_compiled, filename) or re.match(
+                        stockprices_pattern_compiled, filename
+                    ):
+                        self.all_files.append(os.path.join(remote_directory, filename))
+                        sftp.get(filename, os.path.join(local_directory, filename))
+                        logger.info(f"File {filename} downloaded successfully.")
```

Comment:
I think we should log this at debug.
```suggestion
                        logger.debug(f"{filename} downloaded successfully.")
```